controllers/reef_intake.py

```diff
@@ -39,6 +44,18 @@ def intaking(self, initial_call: bool):
 
         self.algae_manipulator_component.intake()
 
+    @state(must_finish=True)
+    def safing(self, initial_call: bool):
+        if initial_call:
+            self.origin_robot_pose = self.chassis.get_pose()
+        robot_pose = self.chassis.get_pose()
+
+        distance = self.origin_robot_pose.translation() - robot_pose.translation()
+
+        if distance.norm() >= self.RETREAT_DISTANCE:
+            self.done()
+            return
```

Comment:
nit: This is the last statement of the state, so we don't need an explicit return here.
```suggestion
```